src/ansys/fluent/core/file_session.py

```diff
@@ -537,6 +586,11 @@ def get_pathlines_field_data(
         Dict
             Dictionary containing a map of surface IDs to the pathline data.
             For example, pathlines connectivity, vertices, and field.
+
+        Raises
+        ------
+        RuntimeError
+            Always, as pathlines are not supported.
```

Comment:
Why do we even have this function if the only thing it does is raise an exception? 🤔 Do we plan to support this later on and this is a placeholder for now?

Reply:
@raph-luc it's not obvious, but the reason is that here we have (re)implemented an established interface. An implementation already existed for live data (in the application). This newer implementation is for file-based data, where the particular data is unavailable.